app/domain/authentication/authn_jwt/authenticator.rb

```diff
@@ -25,6 +25,8 @@ def call
       def create_authentication_parameters
         @logger.debug(LogMessages::Authentication::AuthnJwt::CREATING_AUTHENTICATION_PARAMETERS_OBJECT.new)
         @authentication_parameters = Authentication::AuthnJwt::AuthenticationParameters.new(@authenticator_input)
+        @logger.debug(LogMessages::Authentication::AuthnJwt::FetchingTokenString.new)
+        @authentication_parameters.jwt_token = @jwt_configuration.create_authentication_parameters(@authentication_parameters)
```

Comment:
this is quite odd. to create the `jwt_token` object that is part of the `authentication_parameters` object, you need to `authentication_parameters` object. do we have a more sain way to do it?

Reply:
I'm agree. 
I'm planning to have a way in one of the next PRs